### PR TITLE
New device_platform: serial (supports Teddy Ruxpin)

### PR DESCRIPTION
**Pull request by renekliment**
**Description:**
This can be used for device that uses Arduino for example - like the Teddy Ruxpin project Tedlexa.

**Checklist:**
  - [x] Update CHANGELOG.md
  - [ ] Update Wiki (new platform, installation, configuration changes etc)
  - [x] PR Location (usually `master` for hotfixes, `dev` for general development)
  - [x] need to rename platform files to somethingplatform.py, so this can be renamed to serialplatform.py and it's not serial.py which breaks imports
  - [x] uses unmerged PRs - triggering, logging, therefor fails tests, rebase after that is merged
  - [x] hasn't been tested at all - do testing

@packetrat We'll support bears by default, so you don't have to worry about keeping the code compatible as it gets updated!

### CODE REVIEW

**Comment by ImEmJay** [comment]
I see I'm not the only one who really liked it. And good that I didn't had
the time to start implementing the serial platform for him today.

René Kliment <notifications@github.com> schrieb am Fr., 6. Jan. 2017, 19:51:

File: src/dev-requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+pylint
 fakeRPiGPIO
 -e git+https://github.com/xtacocorex/CHIP_IO.git#egg=CHIP_IO
-pylint
+pyserial
```
Comment (lechat) [inline src/dev-requirements.txt]: This should be in `requirements.txt`

Comment (renekliment) [inline src/dev-requirements.txt]: This is for CI. For users, it gets installed in setup, if they select proper device.

**Comment by renekliment** [comment]
Rebased, fixed and tested with a simple Arduino code (will post later, probably a new alexa-pi organization repo).

@alexa-pi/team Please have a look, this should be easy.

@packetrat Please test. We will roll out a new version soon and it would be awesome, if we could guarantee 100% bear compatibility :-)

**Comment by renekliment** [comment]
The example piece of code can be found here: https://github.com/alexa-pi/AlexaPi-Arduino Contributions are welcome there - it's also an open project. It's main purpose is however the testing of this serial platform and as a pointer for beginners.

**Comment by renekliment** [comment]
14-day PR review window rule ... merging.

@packetrat Please test the `dev` branch now to make sure the bear is supported in the new version.

@alexa-pi/team Only 1 more PR to go to _1.4_. Please have a look at it (#169), so we don't have to wait until Friday for the 14 days to run out. Thank you!

**Pull request merged**